string name
geometry_msgs/Point position
bool in_drawer
bool in_box
bool in_gripper
bool occluded